### PR TITLE
Bias Layer: Add support for regularizer and constraint

### PR DESCRIPTION
**Pull request by saisua**
Small patch to add the bias_regularizer and bias_constraint. It is needed to be like other layers in TF2.0.
They are initialized as None based on current master.

### CODE REVIEW

File: tf_agents/keras_layers/bias_layer.py [hunk 1]
```diff
@@ -39,12 +37,14 @@ class BiasLayer(tf.keras.layers.Layer):
         shape `(batch_size, input_dim)`.
   """
 
-  def __init__(self, bias_initializer='zeros', **kwargs):
+  def __init__(self, bias_initializer='zeros', bias_regularizer=None, bias_constraint=None, **kwargs):
```
Comment (sguada) [inline tf_agents/keras_layers/bias_layer.py]: Can you reformat so it doesn't go over 80 characters and each argument use one line. Ex:
def __init__(self, 
                     bias_initializer='zeros',
                     bias_regularizer=None,
                     bias_constraint=None, 
                     **kwargs):

**Comment by saisua** [comment]
I just did that. Thank you for your quick response.
As a side note, I have seen that there is a file called something like "bias layer test". I have not touched it but feel free to ask for anything.

**Comment by sguada** [comment]
Please add 2 new test to make sure the new arguments work as expected.

**Review by sguada — CHANGES_REQUESTED** [review]
Add tests for the new parameters.

**Comment by tfboyd** [comment]
This has been about 1.5 years. I am making this closed.  We can always reopen.  I am merging PRs where with my knowledge can get the merged. This one is too big for me to handle unfortunately. Sometimes things rot on the vine; sad and a reality. 

Thank you for the effort and sorry this did not make it to the finish.

**Pull request closed without merging**